keps/1-kubernetes-enhancement-proposal-process.md

```diff
@@ -1,21 +1,11 @@
-# Kubernetes Enhancement Proposal Process
-
-## Metadata
```

Comment:
nit: I don't think we should format the Metadata so it's a Markdown table because it can require horizontal scrolling in many cases when rendered and doesn't really make it all that much easier to extract with automation

Reply:
I'm in the process of working with contribx to create a website (kep.k8s.io or community.k8s.io) so that we can present this stuff in a more accessible way. Having the metadata in a form that is compatible with static site generators will make this much easier.

Reply:
Could you explain a bit more about how the format makes it easier to work with static site generators? I do think GitHub is _also_ an entirely reasonable way to interact with KEPs (especially given how few of them we have). I played around with a hacky sort of way to extract an arbitrary block of text between two non rendering Markdown lines using widely available tools (awk) but I need to throw that up as a gist

Reply:
Looking at something like hugo or jekyll, they have an idea of "front matter" that can be used when generating sites based on a pile of markdown.  https://gohugo.io/content-management/front-matter/